common/api/endpoint/ApiEndPoint.ts

```diff
@@ -176,4 +176,10 @@ export const ApiEndPoint = {
     getAll: (countryIso = ':countryIso') => apiPath('userChat', countryIso, 'message', 'all'),
     getNew: (countryIso = ':countryIso') => apiPath('userChat', countryIso, 'message', 'new'),
   },
+  MessageCenter: {
+    topic: {
+      get: () => apiPath('topic'),
+      addMessage: () => apiPath('topic', 'message'),
```

Comment:
Should it be 
```
    topic: {
      get: () => apiPath('topic'),
      addMessage: () => apiPath('topic', 'message'),
```

=>

```
    Topic: {
      one: () => apiPath('topic')
```

with usage

`express.post(MessageCenter.Topic.one(), hanndler)`
`express.get(MessageCenter.Topic.one(), handler)`

Reply:
the requests above are correct. 
express.get(MessageCenter.Topic.one(), handler) will get a topic with messages. 
express.post(MessageCenter.Topic. addMessage(), hanndler) will post a message, not a topic